### DISCUSSION TITLE
How to make `deno fmt` to ignore Markdown files

### DISCUSSION THREAD

**Discussion by dahlia** [denoland/deno#10185] [Q&A] [answered]
Is there a way to make `deno fmt` to ignore Markdown files, in a similar way TypeScript files start with a comment `// deno-fmt-ignore-file`?  I tried to insert `<!-- deno-fmt-ignore-file -->` at the top of a Markdown file, it didn't work.

**Comment by kitsonk** [comment, marked as answer]
Other than excluding it from the glob pattern passed on the command line, no. Adding the ability to add a comments seems like a good idea. Thoughts @dsherret?

**Reply by bartlomieju** [reply]
Looking at our config, it seems we already support ignore directives:
```
.ignore_directive("deno-fmt-ignore")
.ignore_start_directive("deno-fmt-ignore-start")
.ignore_end_directive("deno-fmt-ignore-end")
```
I'm not entirely sure if the first one is used for ignoring the whole file - if so we should change it to `deno-fmt-ignore` to match directive for JS/TS formatter

**Reply by dsherret** [reply]
We have `deno-fmt-ignore-file` for JS, I just forgot to also add it for markdown formatting. Opened https://github.com/dprint/dprint-plugin-markdown/issues/40
